.tests/integration/config.yaml

```diff
@@ -20,16 +20,16 @@ mutect2_gvcf:
   container: "docker://hydragenetics/gatk4:4.1.9.0"
 
 merge_vcf:
-  container: "docker://hydragenetics/common:0.0.1"
+  container: "docker://hydragenetics/common:0.1.0"
```

Comment:
I thought we remove the `container` stanza if it uses the `default_container`?

Reply:
yes we did! But I was thinking that we could do a clean up in a coming commit! These changes are just suppose  to apply the updates for units (and new hydra version) to make it possible to run the develop branch, since the changes will break the any pipelines using develop.

Reply:
Sure, let's skip it then